### PR TITLE
Add goto command and logger

### PR DESCRIPTION
**Pull request by alexhermida**

### CODE REVIEW

File: commander/boat.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        self.connection_string = connection_string
+        self.baudrate = baudrate
+        self._vehicle = None
+
+    def connect(self, wait_ready=True):
+        self._vehicle = dronekit.connect(self.connection_string,
+                                         baud=self.baudrate,
+                                         wait_ready=wait_ready)
+
+    def set_mode(self, mode):
```
Comment (jcleira) [inline commander/boat.py]: @alexhermida its hard for me to comment on this, given the lines that i've written, that are zero..

But this `set properties / while loop` funcs are probably generating `race conditions`, as if after being called someone cancel or changes the setting that they are trying to set, they will stuck on the `while not` loop.

Probably the responsibility about if the setting changed or not, should be on the caller, we may have a helper like `is_armed` with the loop, being called from the caller just right after `arm`

Comment (alexhermida) [reply commander/boat.py]: You're right. As I've been checking this is the first approach in many scripts and they usually wait with a while function.
I read http://python.dronekit.io/develop/best_practice.html#general-considerations
I'll try to introduce the observers http://python.dronekit.io/develop/best_practice.html#monitor-and-react-to-state-changes

Comment (jcleira) [reply commander/boat.py]: 👍 supercaña

**Comment by daavoo** [comment]
We should merge this, right??

**Pull request merged**